src/metal.cs

```diff
@@ -2845,8 +3095,39 @@ interface MTLArgumentEncoder
 		[Export ("constantDataAtIndex:")]
 		IntPtr GetConstantData (nuint index);
 
-		[NoTV][NoiOS]
+		[NoiOS, NoTV]
+		[Mac (10,14, onlyOn64: true)]
+#if XAMCORE_4_0
+		[Abstract]
+#endif
+		[Export ("setRenderPipelineState:atIndex:")]
+		void SetRenderPipelineState ([NullAllowed] IMTLRenderPipelineState pipeline, nuint index);
```

Comment:
No sure about this, but RenderPipelineState could be avoided in the method name due to the first argument.

Reply:
So it'd be `void Set ()`. That's not very C#-y I think, I can't see other cases like this in our bindings also. I think you loose too much information.

Reply:
`Set` is bot very descriptive. Also you risk (we did) running into cases where new API ends up with the same signature as existing one.